src/segment-spinner/stories/index.css

```diff
@@ -0,0 +1,21 @@
+.segment__field {
+  display: flex;
+}
+
+.segment__field--item {
+  padding: 2px;
+  border-radius: 4px;
+}
+.segment__field--item:focus {
+  background-color: #1e65fd;
+  color: white;
+  outline: none;
+}
+
+pre {
+  font-size: 12px;
+  color: rgb(101, 100, 124);
+}
+pre ~ pre {
+  margin-top: 35px;
+}
```

Comment:
Don't style any tag directly, that affects other story aswell.
Add a unique class and target directly.